web_widget_x2many_2d_matrix/README.rst

```diff
@@ -42,7 +42,7 @@ An example use case would be: Select some projects and some employees so that
 a manager can easily fill in the planned_hours for one task per employee. The
 result could look like this:
 
-.. image:: https://raw.githubusercontent.com/web_widget_x2many_2d_matrix/static/description/screenshot.png
+.. image:: /web_widget_x2many_2d_matrix/static/description/screenshot.png
```

Comment:
Please let it download from GH. It makes the link work, no matter where the README is rendered.

The link was wrong, however. Use https://raw.githubusercontent.com/OCA/web/12.0/web_widget_x2many_2d_matrix/static/description/screenshot.png

Reply:
@Yajo, the bot now do this work automatically thanks to https://github.com/OCA/maintainer-tools/pull/397

Reply:
But yes, I think it has to be a bit adjusted for working right now with this automation.